docs/conf.py

```diff
@@ -545,18 +559,18 @@
 ]
 
 rst_epilog = """
-.. |ex| replace:: **Code** 
+.. |ex| replace:: **Example**
 
-.. |out| replace:: **Output** 
+.. |out| replace:: **Result**
```

Comment:
hi @danwos was this change expected? could be the result of a missing rebase?

Reply:
Ha I don't have any skin in the game here. They seem like synonyms to me

Reply:
I used Result instead of Output because it describes what is beneath better than Output.

Also, Code described the content beneath it better than Example.

But all words are synonymous though.

Reply:
I will be sending a PR to update theme's CSS so I can add this change to it.